src/vectorchord/usage/measure-recall.md

```diff
@@ -22,12 +30,114 @@ SELECT vchordrq_evaluate_query_recall(query => $$
+-- schema_name | index_name | table_name | column_name | operator |    value
+---------------+------------+------------+-------------+----------+--------------
+-- public      | idx        | items      | embedding   | <->      | [0.5,0.25,1]
+
+-- List recorded queries for all accessible indexes
+-- DANGER: The result may unexpectedly mix records from multiple indexes. Please check carefully before use. // [!code error]
+-- SELECT COUNT(DISTINCT index_name) FROM vchordrq_sampled_queries WHERE ... // [!code error]
+SELECT * from vchordrq_sampled_queries;
+```
+
+To measure the recall of the recorded queries, you can reconstruct the target query statement. The `vchordrq_sampled_queries` view is **not** recommended here because the evaluated recall will be incorrect if the records come from more than one index.
+
+```sql
+SELECT AVG(recall_value) FROM (
+    SELECT vchordrq_evaluate_query_recall(
+            format(
+                'SELECT ctid FROM %I.%I ORDER BY %I %s %s LIMIT 10',
+                quote_ident(lq.schema_name),
+                quote_ident(lq.table_name),
+                quote_ident(lq.column_name),
+                lq.operator,
+                quote_literal(lq.value)
+            )
+    ) AS recall_value
+    FROM vchordrq_sampled_queries('items_embedding_idx') AS lq
+) AS eval_results;
+```
+
+::: tip IMPORTANT
+- This feature adds an extra 0.3 to 1 millisecond of latency to SELECT statements, which can be reduced by lowering the value of `vchordrq.query_sampling_max_records`.
+- Query records are not synchronized by replication or backup. Each Postgres replica maintains its own set of records based on the queries it processes.
+:::
+
 ## Reference
 
-This feature is not supported by `vchordg`.
+This feature is not supported by `vchordg` and Multi-Vector Retrieval.
+
+### Search Parameters <badge type="info" text="vchordrq" /> {#search-parameters}
+
+#### `vchordrq.query_sampling_enable` <badge type="tip" text="since v0.5.2" />
+
+- Description: Enables or disables the query sampling of `vchordrq` index scan.
+- Type: boolean
+- Default: `off`
+
+#### `vchordrq.query_sampling_max_records` <badge type="tip" text="since v0.5.2" />
+
+- Description: Controls the maximum number of queries recorded per index. This setting is only active when `vchordrq.query_sampling_enable` is on.
+- Type: integer
+- Default: `0`
+- Domain: `[0, 10000]`
+- Example:
+    - `vchordrq.query_sampling_max_records = 0` will maintain 0 queries for each index.
+    - `vchordrq.query_sampling_max_records = 1000` will maintain up to the 1000 most recent queries for each index.
+
+#### `vchordrq.query_sampling_rate` <badge type="tip" text="since v0.5.2" />
+
+- Description: Controls the fraction of vchordrq index scan queries that will be recorded like [log_statement_sample_rate](https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-STATEMENT-SAMPLE-RATE). This setting is only active when `vchordrq.query_sampling_enable` is on.
```

Comment:
1. Two GUCs have nothing in common, except their names. That GUC disables the sampling while this doesn't.
2. Only superusers and users with the appropriate SET privilege can change that GUC. But it's a good idea, we should do the same thing in VectorChord.
3. We'd better clarify that this is random, rather than referring to external documents and let users guess what we want them to look at.